openapi.yml

```diff
@@ -535,19 +535,33 @@ components:
     Offence:
       type: object
       properties:
-        date:
-          type: string
-          format: date
-          example: 1965-12-01
-          description: Date the offence took place or start date if the offence occurred over a period of time
-        code:
+        CJSCode:
```

Comment:
personally unsure whether we'd say CJSCode or cjsCode - probably check w someone else

Reply:
Yes and good spot, that should be a lowercase